pkg/handlers/cluster.go

```diff
@@ -0,0 +1,57 @@
+package handlers
+
+import (
+	"net/http"
+
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	log "github.com/sirupsen/logrus"
+)
+
+type ClusterHandler struct {
```

Comment:
Can we change this to `ClustersServiceHandler` or `ClustersMgmtHandler` to match the ocm service better?

Reply:
@bmeng thanks for reviewing!   
The OCM endpoint this handler deal with is 
`​/api​/clusters_mgmt​/v1​/clusters​/` and the corresponding object is [Cluster](https://pkg.go.dev/github.com/openshift-online/ocm-sdk-go@v0.1.382/clustersmgmt/v1#Cluster), so I picked `Cluster` to make it consistent as the name because the `UpgradePolicy` handler is dealing with the object `UpgradePolicy`.

Do you think we can keep it `ClusterHandler` or prefer `ClustersServiceHandler` ?

Reply:
`ClusterObjectHandler` could work too?